### PR TITLE
Return the sandbox image status as "Pinned"

### PR DESCRIPTION
**Pull request by afbjorklund**
* #39

Upgrade CRI API from v0.22.8 to v0.23.0 in order to get "pinned"

TODO: PodSandboxStats
TODO: ListPodSandboxStats

Upgrade default sandbox image to `registry.k8s.io/pause:3.9`

----

Review

commit 72cfd7cb3a0296f19ad8fb276f87e3885a76c04a

### CODE REVIEW

**Comment by afbjorklund** [comment]
The tests are failing due to known issues with go1.19.11, while the 0.3.4 release used version go1.19.10

File: core/stats.go [hunk 1]
```diff
@@ -78,3 +78,22 @@ func (ds *dockerService) ListContainerStats(
 
 	return &runtimeapi.ListContainerStatsResponse{Stats: stats}, nil
 }
+
+// PodSandboxStats returns stats for a pod sandbox stats request
+func (ds *dockerService) PodSandboxStats(
+	_ context.Context,
+	r *runtimeapi.PodSandboxStatsRequest,
+) (*runtimeapi.PodSandboxStatsResponse, error) {
+	// TODO
```
Comment (nwneisen) [inline core/stats.go:87]: Can we implement this `TODO` since it is done in the alpha version? Or are the same methods not available?

Comment (afbjorklund) [reply core/stats.go:87]: The alpha just "translates" the API between v1 and v1alpha2. It is the same (unimplemented) methods.

**Comment by nwneisen** [comment]
@afbjorklund Can I have you rebase again now that the unit tests are fixed and running in CI?

**Pull request merged**